schemas/documents/contributions/starter.js

```diff
@@ -10,130 +9,53 @@ import {
-          <a href="https://www.sanity.io/docs/starter-templates" target="_blank">
-            starter templates
-          </a>{' '}
-          documentation.
-        </p>
-        <p>
-          If your contribution cannot meet these guidelines, that's OK! You can add it as a showcase
-          project by clicking on the{' '}
-          <a href="/desk/contribution.showcaseProject" target="_blank">
-            "Project for the showcase" item
-          </a>{' '}
-          in the main desk menu of this studio.
-        </p>
-
-        {(!nameValidity || manifestValidity === false) && !deploymentType === 'vercel' && (
-          <div>
-            <h3>Error(s) we spotted with your starter:</h3>
-            <ul>
-              {!nameValidity && (
-                <li>
-                  Repository name doesn't start with <code>sanity-template</code>. Please use the
-                  format <code>{'{owner}/sanity-template-{name}'}</code> (
-                  <a
-                    href="https://www.sanity.io/docs/starter-templates#3a1ad2e88585"
-                    target="_blank"
-                  >
-                    more info
-                  </a>
-                  )
-                </li>
-              )}
-              {manifestValidity === false && (
-                <li>
-                  Sanity.io/create couldn't validate your template. Refer to
-                  <a href="https://www.sanity.io/docs/starter-templates" target="_blank">
-                    the starter templates documentation.
-                  </a>
-                </li>
-              )}
-            </ul>
-          </div>
-        )}
-
-        <h2>Deploying with Vercel Deploy Button</h2>
-        <div>
-          <p>
-            If you are using Vercel, you can use the{' '}
-            <a href="https://vercel.com/docs/deploy-button#generate-your-own" target="_blank">
-              generate your own deploy button
-            </a>{' '}
-            documentation to generate the deployment URL based off your GitHub repository.
-          </p>
-          <p>
-            Once generated, the URL can be copied from the{' '}
-            <a href="https://vercel.com/docs/deploy-button#snippets">
-              Snippets section (by picking the URL tab)
-            </a>
-          </p>
-        </div>
-      </div>
-    );
-  }
-}
-
 export default {
-  title: 'Starter',
+  title: 'Templates',
   name: 'contribution.starter',
   type: 'document',
   icon: RocketIcon,
   initialValue: contributionInitialValue,
   fields: [
+    {
+      name: 'studioVersion',
+      title: 'Studio version',
+      type: 'number',
+      description: 'What Sanity Studio version was this starter was built for.',
+      initialValue: 3,
+      options: {
+        layout: 'radio',
+        direction: 'horizontal',
+        list: [
+          {value: 3, title: 'Studio v3'},
+          {value: 2, title: 'Studio v2'},
+          {value: -1, title: 'n/a'},
```

Comment:
While looking at this I was wondering if it wouldn't make sense to also add the deprecated to the field directly. What do you think @mariuslundgard 
```suggestion
          {value: 2, title: 'Studio v2 (deprecated)'},
          {value: -1, title: 'n/a (deprecated)'},
```